docs/DeveloperGuide.md

```diff
@@ -8,6 +8,17 @@
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+### Study Manager component
+API: `StudyManager.java`
+
+![](images/StudyManagerClass.png)
+
+The `StudyManager` component,
+1. Stores the academic schedule, i.e. all `Module` objects are contained in a `modulesList` object.
+2. The storage can be bound to `modulesList` list such that everytime a change is observed in the list it is saved on to a file.
+3. Does not depend on other components like `ExpenseTracker` and `ContactsManager`
+
```

Comment:
Perhaps a sequence diagram could be included to further elaborate on the operational design of the StudyManager component.